tests/integ/test_deepcopy.py

```diff
@@ -50,6 +52,57 @@
 ]
 
 
+def create_df_with_deep_nested_with_column_dependencies(
+    session, temp_table_name, nest_level: int
+) -> DataFrame:
+    """
```

Comment:
we could add it to test utils but this is also fine.

Reply:
i was actually trying to see if there is such file, but didn't find it. did i miss it somewhere?